WcaOnRails/app/views/competitions_mailer/results_submitted.erb

```diff
@@ -0,0 +1,7 @@
+<p>Results Team,</p>
+
+<p><%= @sender_name %> has uploaded the results for <%= @competition.name %>. You can upload them <a href=<%= "https://www.worldcubeassociation.org/results/admin/upload_results.php?competitionId=#{@competition.id}" %>>here</a>.</p>
```

Comment:
Nit: this will be a bit cleaner if we use `link_to` here instead:

```ruby
link_to "here", "https://www.worldcubeassociation.org/results/admin/upload_results.php?competitionId=#{@competition.id}"
```